tests/gordo_components/builder/test_builder.py

```diff
@@ -519,45 +532,19 @@ def test_provide_saved_model_caching(
         replace_cache=replace_cache,
     )
 
-    model1_creation_date = first_metadata["model"]["model-creation-date"]
-    model2_creation_date = second_metadata["model"]["model-creation-date"]
+    model1_creation_date = first_metadata["metadata"]["build-metadata"]["model"][
+        "model-creation-date"
+    ]
+    model2_creation_date = second_metadata["metadata"]["build-metadata"]["model"][
+        "model-creation-date"
+    ]
     if should_be_equal:
         assert model1_creation_date == model2_creation_date
     else:
         assert model1_creation_date != model2_creation_date
 
-
-@pytest.mark.parametrize(
-    "should_be_equal,evaluation_config",
-    [(True, {"cv_mode": "full_build"}), (False, {"cv_mode": "cross_val_only"})],
-)
-def test_model_builder_cv_scores_only(should_be_equal: bool, evaluation_config: dict):
-    """
-    Test checks that the model is None if cross_val_only is used as the cv_mode.
-    If the default mode ('full_build') is used, the model should not be None.
-
-    Parameters
-    ----------
-    should_be_equal: bool
-        Refers to whether or not the cv_mode should be equal to full (default) or cross_val only.
-    evaluation_config: dict
-        The mode which is tested from within the evaluation_config, is either full or cross_val_only
-
-    """
-
-    model_config = {"sklearn.decomposition.pca.PCA": {"svd_solver": "auto"}}
-    data_config = get_random_data()
-
-    model, metadata = ModelBuilder(
-        name="model-name",
-        model_config=model_config,
-        data_config=data_config,
-        evaluation_config=evaluation_config,
-    ).build()
-    if should_be_equal:
-        assert model is not None
-    else:
-        assert model is None
+    if metadata is not None:
+        assert str(metadata) in str(second_metadata)
```

Comment:
Can you, to make me feel just a tiny bit safer, have this test that `metadata` is at the expected place in `second_metadata`? Which I guess is `second_metadata["metadata"]["user-metadata"]` ?

Reply:
Safety third!